source/whats-new.txt

```diff
@@ -20,10 +20,25 @@ What's New
 Learn about new features, improvements, and fixes introduced in the
 following versions of the {+driver-long+}:
 
+* :ref:`Version 5.2 <scala-version-5.2>`
 * :ref:`Version 5.1.3 <scala-version-5.1.3>`
 * :ref:`Version 5.1.1 <scala-version-5.1.1>`
 * :ref:`Version 5.1 <scala-version-5.1>`
 
+.. _scala-version-5.2:
+
+What's New in 5.2
+-----------------
+
+New features of the 5.2 driver release include:
+
+.. sharedinclude:: dbx/jvm/v5.2-wn-items.rst
+
+   .. replacement:: avs-index-link
+
+      the `SearchIndexModel <{+core-api+}/com/mongodb/client/model/SearchIndexModel.html>`__
```

Comment:
S: sort of a nit, and maybe outside the scope of this PR, but most What's New pages say "Adds..." instead of "Creates..."

np if you don't want to change

Reply:
I can change that in the shared file, good point!